server/base/src/test/java/org/apache/accumulo/server/security/handler/ZKAuthenticatorTest.java

```diff
@@ -88,14 +90,40 @@ public void testTableConversion() {
 
   @Test
   public void testEncryption() {
+    byte[] rawPass = "myPassword".getBytes(Charset.forName("UTF-8"));
+    byte[] storedBytes;
+    try {
+      storedBytes = ZKSecurityTool.createPass(rawPass.clone());
+      assertTrue(ZKSecurityTool.checkCryptPass(rawPass.clone(), storedBytes));
+    } catch (AccumuloException e) {
+      log.error("{}", e.getMessage(), e);
+      fail();
+    }
```

Comment:
You can simplify this JUnit test by just throwing this out of the method. JUnit will show the details of the exception in its logs.

Reply:
Thank you for the suggestion!